charms/tensorboard-controller/requirements-integration.in

```diff
@@ -1,7 +1,8 @@
 # Pinning to <3.0 to ensure compatibility with the 2.9 controller version
 # Note: 3.0 is not being maintained anymore
 juju<3.0
+lightkube
 pytest-operator
+pyyaml
 tenacity
 websockets
```

Comment:
This was my addition, but I can't really tell why it's needed now 😂

Reply:
Well, it is imported and used in `tensorboard-controller/tests/integration/test_charm.py`, what do you mean you don't know?

Reply:
Are you talking about `websockets`?

Reply:
Oh, shite, I thought of pyyaml (as is the shown change), sorry, we should've clarified from the beginning. I can remove it if it's not needed.